bin/crew

```diff
@@ -241,17 +241,17 @@ end
 
 def list_installed
   if @opt_verbose
-    @installed_packages = []
+    installed_packages = []
     @device[:installed_packages].each do |package|
       search package[:name], true
-      @installed_packages.append("#{package[:name]} #{package[:version]}")
+      installed_packages.append("#{package[:name]} #{package[:version]}")
     end
-    @sorted_installed_packages = @installed_packages.sort
-    @sorted_installed_packages.unshift('======= =======')
-    @sorted_installed_packages.unshift('Package Version')
-    @first_col_width = @sorted_installed_packages.map(&:split).map(&:first).max_by(&:size).size + 2
-    @sorted_installed_packages.map(&:strip).each do |line|
-      puts "%-#{@first_col_width}s%s".lightgreen % line.split
+    sorted_installed_packages = installed_packages.sort
+    sorted_installed_packages.unshift('======= =======')
+    sorted_installed_packages.unshift('Package Version')
+    first_col_width = sorted_installed_packages.map(&:split).map(&:first).max_by(&:size).size + 2
+    sorted_installed_packages.map(&:strip).each do |line|
+      puts "%-#{first_col_width}s%s".lightgreen % line.split
     end
```

Comment:
You don't need `sorted_installed_packages` with `installed_packages.sort!`.  For example:
```
irb(main):001> lcm = ['larry','curly','moe']
=> ["larry", "curly", "moe"]
irb(main):002> lcm.sort
=> ["curly", "larry", "moe"]
irb(main):004> puts lcm.to_s
["larry", "curly", "moe"]
irb(main):005> lcm.sort!
=> ["curly", "larry", "moe"]
irb(main):006> puts lcm.to_s
["curly", "larry", "moe"]
```

Reply:
Good point! I'll put that in the next PR, so that this one stays as exclusively demoting variable scope.